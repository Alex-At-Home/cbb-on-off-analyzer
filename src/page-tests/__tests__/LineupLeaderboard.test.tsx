import React from 'react';
import LineupLeaderboardPage from '../../pages/LineupLeaderboard';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe("LineupLeaderboardPage", () => {
  test("LineupLeaderboardPage - should create snapshot", () => {
    //TODO: this isn't the best test because it will just step over the async fetch
    //and render a trivial page, still will do for now
    
    const wrapper = shallow(<LineupLeaderboardPage testMode={true}/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
