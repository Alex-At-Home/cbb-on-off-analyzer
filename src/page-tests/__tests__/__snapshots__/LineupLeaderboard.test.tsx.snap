// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LineupLeaderboardPage LineupLeaderboardPage - should create snapshot 1`] = `
<Container
  className="medium_screen"
  fluid={false}
>
  <SiteModeDropdown />
  <Row
    className="mt-2"
    noGutters={false}
  >
    <Col
      className="text-center"
      xs={12}
    >
      <LandingPageIcon />
      <h3>
        Lineup Leaderboard
      </h3>
    </Col>
  </Row>
  <Row
    className="border-bottom"
    noGutters={false}
  >
    <HeaderBar
      common={
        Object {
          "gender": "",
          "year": "",
        }
      }
      thisPage="lineup-_leaderboard"
    />
  </Row>
  <Row
    className="mt-3"
    noGutters={false}
  >
    <LineupLeaderboardTable
      dataEvent={
        Object {
          "confs": Array [],
          "lastUpdated": 0,
          "lineups": Array [],
        }
      }
      onChangeState={[Function]}
      startingState={Object {}}
    />
  </Row>
  <Footer
    dateOverride={0}
    server="server"
  />
</Container>
`;
